test/models.test.js

```diff
@@ -1,59 +1,227 @@
+      var model = new models.Centre();
+      var data = [{
+        type: 'male',
+        capacity: 100,
+        occupied: 40,
+        ooc: 5
+      }];
+
+      model.on('change', function() {
+        var male_beds = _.find(model.get('beds'), function(item) {
+          return item.type === "male"
+        });
+
+        expect(male_beds.available).to.be(55);
+        return done();
+      });
 
-      expect(collection).to.have.length(1);
+      model.set('beds',data);
     });
-    it('should update an existing model', function() {
-      var centre = new models.Centre();
-      var collection = new collections.Centres(centre);
 
-      collection.add(new models.Centre());
+    it('should check whether there is only 1 male bed', function(done) {
+      var model = new models.Centre();
+      var data = [{
+        type: 'male',
+        capacity: 100,
+        occupied: 95,
+        ooc: 4
+      }];
 
-      expect(collection).to.have.length(2);
+      model.on('change', function() {
+        var male_beds = _.find(model.get('beds'), function(item) {
+          return item.type === "male"
+        });
 
-      collection.add(centre);
+        expect(male_beds.available).to.be(1);
+        expect(male_beds.plurality).to.be("");
+        return done();
+      });
 
-      expect(collection).to.have.length(2);
+      model.set('beds',data);
     });
-    it('should remove a model', function() {
-      var centre = new models.Centre();
-      var collection = new collections.Centres(centre);
 
-      collection.remove(centre);
+    it('should check whether there are only 2 male beds', function(done) {
+      var model = new models.Centre();
+      var data = [{
+        type: 'male',
+        capacity: 100,
+        occupied: 95,
+        ooc: 3
+      }];
+
+      model.on('change', function() {
+        var male_beds = _.find(model.get('beds'), function(item) {
+          return item.type === "male"
+        });
 
-      expect(collection).to.have.length(0);
+        expect(male_beds.available).to.be(2);
+        expect(male_beds.plurality).to.be("s");
+        return done();
+      });
+
+      model.set('beds',data);
     });
-  });
-});
 
-describe('models', function() {
-  describe('centre', function() {
-    it('should be instantiated', function() {
+    it('should calculate female available beds', function(done) {
+      var model = new models.Centre();
+      var data = [{
+        type: 'female',
+        capacity: 100,
+        occupied: 40,
+        ooc: 5
+      }];
+
+      model.on('change', function() {
+        var female_beds = _.find(model.get('beds'), function(item) {
+          return item.type === "female"
+        });
+
+        expect(female_beds.available).to.be(55);
+        return done();
+      });
+
+      model.set('beds',data);
+    });
+
+    it('should check whether there is only 1 female bed', function(done) {
+      var model = new models.Centre();
```

Comment:
you could move all the `model` and `data` definitions into a `beforeEach()` which would reduce the repetition between your tests here.